crates/sequencer_proof_client/src/file_based_proof_client.rs

```diff
@@ -144,3 +145,93 @@ impl ProofClient for FileBasedProofClient {
         Ok(())
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::{sequencer_proof_client::SequencerProofClient, PeekableProofClient};
+
+    #[tokio::test]
```

Comment:
We definitely should enable the tests. Not necessarily as part of this PR, but long term.